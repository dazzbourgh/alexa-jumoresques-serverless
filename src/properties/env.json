{
  "default": {
    "vk": {
      "accessToken": "#arn:aws:secretsmanager:us-west-1:403020287921:secret:vk-BcAwdu.accessToken",
      "apiVersion": "5.103",
      "domain": "jumoreski"
    },
    "aws": {
      "polly": {
        "voice": "Maxim",
        "outputFormat": "mp3"
      },
      "s3": {
        "bucketName": "jumoresques",
        "key": "jumoresques.mp3"
      },
      "accessKeyId": "#aws.accessKeyId",
      "accessKey": "#aws.accessKey"
    }
  },
  "production": {
    "aws":{
      "s3": {
        "region": "us-west-1"
      }
    }
  }
}
