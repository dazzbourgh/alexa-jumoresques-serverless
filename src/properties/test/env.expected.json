{
  "vk": {
    "token": "vkToken",
    "domain": "abc"
  },
  "aws": {
    "token": "defaultVal",
    "bucket": "someVal"
  }
}
